Emby.Dlna/PlayTo/DlnaHttpClient.cs

```diff
@@ -54,15 +56,34 @@ private static string NormalizeServiceUrl(string baseUrl, string serviceUrl)
                     LoadOptions.None,
                     cancellationToken).ConfigureAwait(false);
             }
-            catch (XmlException ex)
+            catch (XmlException)
             {
-                _logger.LogError(ex, "Failed to parse response");
-                if (_logger.IsEnabled(LogLevel.Debug))
+                // try correcting the Xml response with common errors
+                var xmlString = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
+
+                // find and replace unescaped ampersands (&)
+                Regex regex = new Regex(@"(&(?![a-z]*;))");
```

Comment:
Very inefficient. Either put it in a static variable or use the static methods i.e. `Regex.Replace(pattern, string)`

Reply:
Agreed. Replaced with the new GeneratedRegex.